### DISCUSSION TITLE
How to set state on CTkSegmentedButton (separately)

### DISCUSSION THREAD

**Discussion by felle9900** [TomSchimansky/CustomTkinter#1297] [General]
I have a separately with 3 "buttons".

how do I individually set the state on them ?

I usually use: "button.configure(state="disabled")"

But its not all the 3 sub buttons i want to disable/enable...
Is this possible ?

**Comment by felle9900** [comment]
I found a solution on my problem:
self.seg_btn_tag_options._buttons_dict["Delete"].configure(state="disabled")

**Comment by Wolf-SO** [comment]
@TomSchimansky can this be moved to the Q&A section?
